libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -1114,6 +1114,17 @@ void AC_AttitudeControl::input_shaping_rate_predictor(const Vector2f &error_angl
     target_ang_vel_rads.y = ang_vel_rads.y;
 }
 
+// scale I to represent the current angle P
+void AC_AttitudeControl::scale_I_to_angle_P()
+{
+    Vector3f i_scale{
+        _p_angle_roll.kP() * _angle_P_scale.x,
+        _p_angle_pitch.kP() * _angle_P_scale.y,
+        _p_angle_yaw.kP() * _angle_P_scale.y
```

Comment:
Pitch and yaw get the same scale? Shouldn't this be
_p_angle_yaw.kP() * _angle_P_scale.z ?